src/math/p5.Vector.js

```diff
@@ -1636,6 +1771,32 @@ p5.Vector.add = function add(v1, v2, target) {
   return target;
 };
 
+// Returns a vector remainder when it is divided by another vector
+/**
+ * @method rem
+ * @static
+ * @param  {p5.Vector} v1 dividend <a href="#/p5.Vector">p5.Vector</a>
+ * @param  {p5.Vector} v2 divisor <a href="#/p5.Vector">p5.Vector</a>
+ * @param  {p5.Vector} target the vector to receive the result
+ */
+/**
+ * @method rem
+ * @static
+ * @param  {p5.Vector} v1
+ * @param  {p5.Vector} v2
+ * @return {p5.Vector} the resulting <a href="#/p5.Vector">p5.Vector</a>
+ *
+ */
+p5.Vector.rem = function rem(v1, v2, target) {
```

Comment:
The function itself doesn't need to be named, especially considering another member function above have the same name as well so even for debugging it isn't especially useful.

Reply:
Just noticed member functions in p5.Vector are all named while others throughout the codebase aren't. I think you can leave this for now as this seems to be a separate issue of its own.